### DISCUSSION TITLE
svelte sdk, new version

### DISCUSSION THREAD

**Discussion by stepro71** [appwrite/appwrite#1633] [Q&A] [answered]
will the svelte sdk be updated, e.g for the realtime api?

**Comment by TorstenDittmann** [comment, marked as answer]
Hey @stepro71 👋🏻 

Yes, we are aware of that the Svelte SDK is outdated. This needs a bit more planning, now that Svelte Kit is now a thing. Previously the SDK was planned as a Wrapper for the Client Side and native Svelte. For Svelte Kit we need to consider the server side as well - which is still not decided when we do that.

For now please use the normal Web SDK 🙏🏻

**Reply by eldadfux** [reply]
@TorstenDittmann would you like to test the idea of moving the Svelte SDK to the generator? If it works well, maybe we can do the same with other frameworks.

**Reply by TorstenDittmann** [reply]
We could try, but such a library might not make that much sense anymore with Svelte Kit which introduces the option of SSR.
